apps/stress-test/webpack.config.js

```diff
@@ -0,0 +1,130 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const TerserWebpackPlugin = require('terser-webpack-plugin');
+const { GriffelCSSExtractionPlugin } = require('@griffel/webpack-extraction-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+const isProd = process.env.NODE_ENV === 'production';
+
+const pages = [
+  {
+    name: 'v8-simple-stress',
+    path: './src/v8/simple-stress/index.tsx',
+    template: './src/v8/simple-stress/index.html',
+    output: 'v8/simple-stress/index.html',
+  },
+  {
+    name: 'v9-simple-stress',
+    path: './src/v9/simple-stress/index.tsx',
+    template: './src/v9/simple-stress/index.html',
+    output: 'v9/simple-stress/index.html',
+  },
+  {
+    name: 'wc-simple-stress',
+    path: './src/wc/simple-stress/index.wc.ts',
+    template: './src/wc/simple-stress/index.html',
+    output: 'wc/simple-stress/index.html',
+  },
+];
+
+const config = {
+  mode: isProd ? 'production' : 'development',
+  entry: pages.reduce((config, page) => {
+    config[page.name] = page.path;
+    return config;
+  }, {}),
+  output: {
+    filename: '[name].[contenthash].bundle.js',
+    sourceMapFilename: '[name].[contenthash].map',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devtool: 'source-map',
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|ts|tsx)$/,
+        // Apply "exclude" only if your dependencies **do not use** Griffel
+        // exclude: /node_modules/,
+        exclude: /\.wc\.(ts|tsx)?$/,
+        use: {
+          loader: GriffelCSSExtractionPlugin.loader,
```

Comment:
the css extraction plugin is still experimental, do we want our perf tests to be based off that ? is it possible to have multiple webpack configs so we can test different parameters i.e. only griffel runtime vs griffel build transform vs griffel css extraction

It would be useful since we'd also know to what extent Griffel affects our components' performance

Reply:
I think it's a good idea to test the different Griffel scenarios. Made a follow up issue: #24310